firmware/ja-rule.X/nbproject/configurations.xml

```diff
@@ -470,7 +602,7 @@
         <targetDevice>PIC32MX675F512H</targetDevice>
         <targetHeader></targetHeader>
         <targetPluginBoard></targetPluginBoard>
-        <platformTool>PICkit3PlatformTool</platformTool>
+        <platformTool>Simulator</platformTool>
```

Comment:
Revert this perhaps? I'm not sure how we'll deal with different people with different programmers anyway.